### PR TITLE
Feature/marketing test enrique

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/salesforce/ui/pages/oportunitieslist/OpportunityListLightningPage.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * @version 1.0 20 March 2020.
+ */
+public class OpportunityListLightningPage extends OpportunityListPageAbstract {
+
+    @FindBy(css = "table[role='grid']")
+    private WebElement opportunityTable;
+
+    private static final String OPPORTUNITY_PARTIAL_LOCATOR = "a[title='%s']";
+    private static final String CELL_ATTRIBUTE = "href";
+
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/oportunitieslist/OpportunityListLightningPage.java:30]: delete this line

File: src/main/java/salesforce/ui/pages/opportunities/OpportunitiesPopUpLightning.java [hunk 2]
```diff
@@ -32,6 +38,17 @@ protected void waitUntilPageObjectIsLoaded() {
         webDriverWait.until(ExpectedConditions.elementToBeClickable(newButton));
     }
 
+    @Override
+    public void displayOpportunityList(final String listName) {
+        String opportunityOrderedListLocator;
+
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/opportunities/OpportunitiesPopUpLightning.java:44]: mayle you can declare this varible out of the method
and delete the 44 ompty line

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 3]
```diff
@@ -13,12 +13,15 @@
...
 import io.cucumber.java.en.When;
+
+import salesforce.ui.pages.AppPageFactory;
 import org.testng.Assert;
 import salesforce.api.requestapi.OpportunityApiHelper;
 import salesforce.entities.Context;
 import salesforce.entities.Opportunity;
 import salesforce.utils.SheetManager;
 import salesforce.ui.pages.PageTransporter;
+
```
Comment (JuanCarlosMartinez4) [inline src/test/java/salesforcetest/steps/OpportunityStep.java]: delete this line

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 4]
```diff
@@ -13,12 +13,15 @@
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+
```
Comment (JuanCarlosMartinez4) [inline src/test/java/salesforcetest/steps/OpportunityStep.java]: delete this line

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 5]
```diff
@@ -77,7 +81,7 @@ public void displaysAnInformationMessageInOpportunityPageWithTheFormat(final Str
     /**
      * Allows to create many opportunities.
      *
-     * @param quantity number of opportunities.
+     * @param quantity        number of opportunities.
```
Comment (JuanCarlosMartinez4) [inline src/test/java/salesforcetest/steps/OpportunityStep.java]: dele the empty space **quantity        number of opportunities.**

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 6]
```diff
@@ -113,9 +117,12 @@ public void navigatesToOpportunitiesPage() {
     /**
      * Search an opportunity.
      *
-     * @param arg0 contains opportunity name.
+     * @param opportunityName contains a String value.
+     * @param listName        contains a String value.
```
Comment (JuanCarlosMartinez4) [inline src/test/java/salesforcetest/steps/OpportunityStep.java]: **listName        contains a String value.** delete empty space.

File: src/main/java/salesforce/ui/pages/opportunities/OpportunitiesPopUpLightning.java (hunk 2 above)
Comment (Enrique-C) [reply src/main/java/salesforce/ui/pages/opportunities/OpportunitiesPopUpLightning.java:44]: Why out, it was not have another call, it is not necessary.

File: src/main/java/salesforce/ui/pages/oportunitieslist/OpportunityListLightningPage.java [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    @Override
+    public void clickOnOpportunity(final String opportunityName) {
+
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/oportunitieslist/OpportunityListLightningPage.java]: this space is not necessary

File: src/test/resources/features/Opportunity.feature [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
-    And I navigate to Opportunities Page
-    And I search an opportunity "OpportunityTest"
-    When I change an opportunity's owner with "CurrentOwner"
-    Then the application should display an information message in Opportunity page with the format "[[Owner Name]] now owns the record for [[Opportunity Name]]"
+#  Scenario: Change opportunity owner to another user
+#    Given I create opportunity as OpportunityTest
+#    And I navigate to Opportunities Page
+#    And I search an opportunity "OpportunityTest" in list "All Opportunities"
+#    When I change an opportunity's owner with "CurrentOwner"
+#    Then the application should display an information message in Opportunity page with the format "[[Owner Name]] now owns the record for [[Opportunity Name]]"
```
Comment (JuanCarlosMartinez4) [inline src/test/resources/features/Opportunity.feature]: uncomment this lines

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 9]
```diff
@@ -13,12 +13,14 @@
...
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import salesforce.ui.pages.AppPageFactory;
 import org.testng.Assert;
 import salesforce.api.requestapi.OpportunityApiHelper;
 import salesforce.entities.Context;
 import salesforce.entities.Opportunity;
 import salesforce.utils.SheetManager;
 import salesforce.ui.pages.PageTransporter;
+
```
Comment (JuanCarlosMartinez4) [inline src/test/java/salesforcetest/steps/OpportunityStep.java]: delete this line

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**